src/services/syncer/strategies/transifex/utils/api.js

```diff
@@ -37,7 +37,20 @@ async function getProject(token, options) {
   const { data } = apiResult.data;
 
   let result = {};
-  if (data[0]) result = data[0].attributes;
+  if (data[0]) {
+    const { attributes, relationships } = data[0];
+    result = attributes;
+    // also extract source language
+    if (relationships
```

Comment:
_Suggestion:_ Nothing wrong with this, but since we are already loading lodash we could use https://lodash.com/docs/4.17.15#get